cmp/grid/Grid.ts

```diff
@@ -322,7 +322,7 @@ export class GridLocalModel extends HoistModel {
     getContextMenuItems = (params: GetContextMenuItemsParams) => {
         const {model} = this,
             {contextMenu} = model;
-        if (!contextMenu || XH.isMobileApp || model.isEditing) return null;
+        if (!contextMenu || XH.isMobileApp || model.isEditing) return [];
```

Comment:
Issue https://github.com/xh/hoist-react/issues/4123

Originates from this change in Ag-Grid: https://github.com/ag-grid/ag-grid/pull/9558/files#diff-ca797f3080ead2f4e3dc3d52703d9696e2e82f94bd2d8c598a564798b0574f5cL117-L121

Reply:
nice fix!